requirements.txt

```diff
@@ -27,9 +27,11 @@ Unidecode==1.2.0
 urllib3==1.26.5
 wheel
 zipp==3.4.0
+black
 -e ./src/core
 -e ./src/zos_console
 -e ./src/zos_files
 -e ./src/zos_jobs
 -e ./src/zos_tso
--e ./src/zosmf
\ No newline at end of file
+-e ./src/zosmf
+black
```

Comment:
Please remove this 2nd instance of "black", we should only need it once in requirements.txt